test cases/unit/73 summary/meson.build

```diff
@@ -8,6 +8,7 @@ summary({'Some boolean': false,
          'Some string': 'Hello World',
          'A list': ['string', 1, true],
          'empty list': [],
+         'enabled_opt': get_option('enabled_opt'),
```

Comment:
There is a unit test that verify the output, you'll see that CI fails now. See test_summary in run_unittests.py

Reply:
What do you think reads better:

- `feature in summary`
- `UserOption` in summary